pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -208,6 +208,9 @@ func TestCouchDBStore_Get(t *testing.T) {
 		value, err := store.Get(testDocKey)
 		require.NoError(t, err)
 		require.Equal(t, testJSONValue, string(value))
+
+		err = store.Put(testDocKey, []byte(testJSONValue))
```

Comment:
Is this testing the update functionality? Don't you want to check and see if the _rev number is updated correctly?

Reply:
yes

Reply:
done